charts/pega/templates/_helpers.tpl

```diff
@@ -227,6 +227,9 @@ true
 {{- define "generatedPodAnnotations" }}
 {{- end }}
 
+{{- define "generatedHazelcastServicePodAnnotations" }}
```

Comment:
Why is this here and not in the hazelcast _helpers.tpl?

Also, are datadog annotations being added later?  They should be defined here, right?

Reply:
I moved the empty definition to hazelcast _helpers.tpl.

They would be defined in the platform infinity SCE specific helm templates . The annotations are for fnx depoyment , so we need to override the empty definition for CMC just like `generatedPodAnnotations`